GOVERNANCE.md

```diff
@@ -0,0 +1,126 @@
+Governance for the R data.table project
+
+# Purpose and scope
+
+## This document
+
+The purpose of this document is to define how people related to the project work together, so that the project can expand to handle more contributions, from a larger and more diverse group of contributors.
```

Comment:
```suggestion
The purpose of this document is to define how people related to the project work together, so that the project can expand to handle a larger and more diverse group of contributors.
```